OpenSim/Common/Test/testComponentInterface.cpp

```diff
@@ -59,10 +59,14 @@ class TheWorld : public Component {
     void add(Component* comp) {
         // add it the property list of components that owns and serializes them
         updProperty_components().adoptAndAppendValue(comp);
+        addComponent(comp);
     }
 
     // Top level connection method for all encompassing Component
-    void connect() { Super::connect(*this); }
+    void connect() { 
+        initComponentTreeTraversal(*this);
+        Super::connect(*this);
```

Comment:
Does this go against the policy that `Super::` calls should be the first in a method?